packages/patternfly-4/react-core/src/components/Select/Select.md

```diff
@@ -325,6 +407,84 @@ class TypeaheadSelectInput extends React.Component {
 }
 ```
 
+## Disabled typeahead select input
+
+```js
+import React from 'react';
+import { Select, SelectOption, SelectVariant } from '@patternfly/react-core';
+
+class TypeaheadSelectInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.options = [
+      { value: 'Alabama', disabled: false },
```

Comment:
I'd remove the disabled keys from the options array in the two added examples for clarity purposes

Reply:
I was thinking they same thing.